web/beacon-app/src/features/onboarding/components/steps/organization/index.tsx

```diff
@@ -1,19 +1,41 @@
 import { Trans } from '@lingui/macro';
+import { useEffect } from 'react';
 
+import { useUpdateMember } from '@/features/members/hooks/useUpdateMember';
 import useUserLoader from '@/features/members/loaders/userLoader';
+import { getOnboardingStepsData } from '@/features/onboarding/shared/utils';
 import { useOrgStore } from '@/store';
 
 import StepCounter from '../StepCounter';
 import OrganizationForm from './form';
 
 const OrganizationStep = () => {
-  const increaseStep = useOrgStore((state: any) => state.increaseStep) as any;
   const { member } = useUserLoader();
+  const { updateMember, wasMemberUpdated, isUpdatingMember, error, reset } = useUpdateMember();
+  const increaseStep = useOrgStore((state: any) => state.increaseStep) as any;
+
+  // Display error if organization name is already taken.
+  const hasError = error && error.response.status === 409;
 
   const submitFormHandler = (values: any) => {
-    console.log(values);
+    const payload = {
+      memberID: member?.id,
+      payload: {
+        ...getOnboardingStepsData(member),
+        organization: values.organization,
+      },
+    };
+    updateMember(payload);
     increaseStep();
```

Comment:
Would you mind removing this?  We should only increase it after the request succeeds. If possible, let's remove it wherever it has been added.

Reply:
Sure. I believe this was there before I merged changes and it got missed while making updates.